### DISCUSSION TITLE
Support distinguishing RUNs

### DISCUSSION THREAD

**Discussion by overlabs** [visualDust/neetbox#80] [Ideas]
# NEETBOX now uses run-id to distinguish runs

in #76 (https://github.com/visualDust/neetbox/pull/76/commits/0a391e57250e5369eac534d730157661a6dbd34f), we added run id columns to history db and server as well as client python api, to distinguish runs. On every run of a neetbox project, neetbox generates a new run id. The original intention of adding this design was due to tensorboard-sometimes users need to view the data generated by each project run separately.
However, we do support static run id. if user add run id to config file `neetbox.toml` generated by cli, the project will use this specific run id on every launch. Besides, users should be able to choose whether to distinguish each run in frontend(frontend has not started on run id yet and we have a lot of freedom about how  to design this feature at user level).

further updates will be posted here

**Comment by visualDust** [comment]
# Update

NEETBOX support run id now!

![image](https://github.com/visualDust/neetbox/assets/33346934/9d21c699-d6ca-49c9-8543-0d0621deb04d)

**Comment by visualDust** [comment]
# update: frontend can manage runs

![image](https://github.com/visualDust/neetbox/assets/33346934/31c277be-016e-43ac-bf87-fcbdca7f5da9)

**Reply by visualDust** [reply]
done in #110
